admin_guide/enabling_EFK.adoc

```diff
@@ -0,0 +1,357 @@
+= Aggregating Logs with the EFK Stack
```

Comment:
Same comment as above wrt to the title, though you have more space in the topic title so "with the EFK Stack" may be fine to keep here.